pyteal/ast/itxn.py

```diff
@@ -200,6 +206,111 @@ def SetFields(cls, fields: Dict[TxnField, Union[Expr, List[Expr]]]) -> Expr:
         fieldsToSet = [cls.SetField(field, value) for field, value in fields.items()]
         return Seq(fieldsToSet)
 
+    @classmethod
+    def MethodCall(
+        cls,
+        *,
+        app_id: Expr,
+        method_signature: str,
+        args: List[Union[abi.BaseType, Expr, Dict[TxnField, Union[Expr, List[Expr]]]]],
+        fields: Dict[TxnField, Union[Expr, List[Expr]]],
+    ) -> Expr:
+
+        # Default, always need these
+        fields_to_set = [
+            cls.SetField(TxnField.type_enum, TxnType.ApplicationCall),
+            cls.SetField(TxnField.application_id, app_id),
+        ]
+
+        # We only care about the args
+        arg_type_specs, _ = type_specs_from_signature(method_signature)
```

Comment:
```suggestion
        (arg_type_specs: list[TypeSpec], _: TypeSpec) = type_specs_from_signature(method_signature)
```
something like this ought to work, and then there'll be consistency with typing local var's

Reply:
Probably need to define on a separate line:

```python
arg_type_specs: list[TypeSpec]
```